src/utils.ts

```diff
@@ -0,0 +1,125 @@
+import { publish as _publish } from 'libnpmpublish'
+import got from 'got'
+import ssri from 'ssri'
+import URL from "url"
+import {MigrationSettings} from "./settings";
+
+const logger = require('pino')()
+
+export function getScopedOptions(packageName: string, source: Record<string, string>) {
+  const scope = getNamedScope(packageName)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263162)
```suggestion
  const scope = getNamedScope(packageName);
```